src/AppInstallerCLI.sln

```diff
@@ -68,6 +66,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Valijson", "Valijson\Valijs
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ManifestSchema", "ManifestSchema\ManifestSchema.vcxitems", "{7D05F64D-CE5A-42AA-A2C1-E91458F061CF}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AppInstallerCLISchemas", "AppInstallerCLISchemas\AppInstallerCLISchemas.vcxitems", "{952B513F-8A00-4D74-9271-925AFB3C6252}"

```

Comment:
>AppInstallerCLISchemas [](start = 79, length = 22)

We are trying to move away from AppInstallerCLI by at least not adding more instances of it.  Use WinGet instead.

Reply:
Renamed it to WinGetSchemas